utils/api/analyzer.go

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
nit: why do you put all these under `utils`?  utils is for shareable code among different packages, looks like we'd better just put it under `openapi/blabla`?

Reply:
Because this package is not only serving for openapi.